README.md

```diff
@@ -1,54 +1,32 @@
-[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0)
-# BHoM
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://www.gnu.org/licenses/lgpl-3.0) [![Build status](https://ci.appveyor.com/api/projects/status/11a3ucotxcl9746k/branch/master?svg=true)](https://ci.appveyor.com/project/BHoMBot/ram-toolkit/branch/master) [![Build status](https://ci.appveyor.com/api/projects/status/r87fym5si9vr5io5/branch/master?svg=true)](https://ci.appveyor.com/api/projects/status/ram_toolkit/branch/master)
```

Comment:
The first build status badge refers to Robot and should be replaced by the second badge which is in the [comment here](https://github.com/BHoM/RAM_Toolkit/issues/143#issuecomment-626582005).

There should also be the badge from Azure here as well.

Reply:
Updated!